apps/console/src/extensions/i18n/resources/en-US/extensions.ts

```diff
@@ -31,7 +31,10 @@ export const extensions: Extensions = {
         help: {
             communityLink: "Ask the Community",
             docSiteLink: "Documentation",
-            helpCenterLink: "Contact Support",
+            helpCenterLink: {
+                title: "Contact Support",
+                subtitle: "Talk to the Asgardeo team to obtain personalized assistance."
```

Comment:
Let's not hardcode any productName. Let's use {{ productName }} and supply the name from config.

Reply:
Fixed with: https://github.com/wso2/identity-apps/pull/6680/commits/a938e76e6c9fbc179ee78185b96e953c5b2a0f3c